packages/@react-spectrum/dropzone/docs/DropZone.mdx

```diff
@@ -0,0 +1,374 @@
+
+```css
+.draggable {
+  display: inline-block;
+  vertical-align: top;
+  border: 1px solid gray;
+  padding: 10px;
+  margin-right: 20px;
+  margin-bottom: 20px;
+  border-radius: 4px;
+  height: fit-content;
+}
+
+.draggable.dragging {
+  opacity: 0.5;
+}
+
+.files{
+  margin-top: 20px;
+}
+```
+
+</details>
+
+
+## Props
+
+<PropTable component={docs.exports.DropZone} links={docs.links} />
+
+## Visual options
+
+### Filled state
+
+The user is responsible for both managing the filled state of a drop zone and handling the associated styling. To set the drop zone to a filled state, the user must pass the `isFilled` prop.
+
+The example below demonstrates one way of styling the filled state.
+
+```tsx example
+function Example() {
+  let [filledSrc, setFilledSrc] = React.useState(null);
+  let [isFilled, setIsFilled] = React.useState(false);
+
+  return (
+    <>
+      <DraggableImage />
+      <DropZone
+        isFilled={isFilled}
+        maxWidth="size-3000"
+        height="size-2400"
+        UNSAFE_className={isFilled && 'is-filled'}
+        getDropOperation={(types) =>  (types.has('image/png') || types.has('image/jpeg')) ? 'copy' : 'cancel'}
+        onDrop={async (e) => {
+          e.items.find(async (item) => {
+            if (item.kind === 'file') {
+              if (item.type === 'image/jpeg' || item.type === 'image/png') {
+                setFilledSrc(URL.createObjectURL(await item.getFile()));
+                setIsFilled(true);
+              }
+            } else if (item.kind === 'text') {
+              setFilledSrc(await item.getText('image/jpeg'));
+              setIsFilled(true);
+            }
+          });
+        }}>
+        <IllustratedMessage>
+            <Upload />
+            <Heading>
+              <Text slot="label">
+                Drag and drop photos
+              </Text>
+            </Heading>
+        </IllustratedMessage>
+        {isFilled && <img className={'images'} alt="" src={filledSrc} />}
+      </DropZone>
+    </>
+  );
+}
+```
+
+<details>
+  <summary style={{fontWeight: 'bold'}}><ChevronRight size="S" /> Show CSS</summary>
+
+```css
+.images{
+  position: absolute;
+  top: 0px;
+  left: 0px;
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+  border-radius: var(--spectrum-alias-border-radius-small);
+}
+
+.is-filled {
+  border-style: solid;
+}
+
+```
+</details>
+
+The `DraggableImage` component used above is defined below. See [useDrag](../react-aria/useDrag.html) for more details and documentation.
+
+<details>
+  <summary style={{fontWeight: 'bold'}}><ChevronRight size="S" /> Show code</summary>
+
+```tsx example render=false export=true
+function DraggableImage() {
+  let {dragProps, isDragging} = useDrag({
+    getItems() {
+      return [
+        {
+          'image/jpeg': 'https://images.unsplash.com/photo-1528722828814-77b9b83aafb2?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxzZWFyY2h8MTF8fHNwYWNlfGVufDB8fDB8fHww&auto=format&fit=crop&w=900&q=60'
```

Comment:
For brevity and consistency with other docs, we may want to use imgur links. There are some already used in the repo, and I bet @snowystinger probably has more if you need them.